packages/rancher-monitoring/rancher-monitoring.patch

```diff
@@ -1,82 +1,60 @@
+-name: kube-prometheus-stack
++  - name: Arvind
++    email: iyengararvind@gmail.com
++name: rancher-monitoring
+ sources:
+   - https://github.com/prometheus-operator/kube-prometheus
+   - https://github.com/prometheus-operator/prometheus-operator
+@@ -22,5 +30,6 @@
+ tillerVersion: ">=2.12.0"
+ home: https://github.com/prometheus-operator/kube-prometheus
  keywords:
 -- operator
 -- prometheus
 +  - operator
 +  - prometheus
 +  - monitoring
- maintainers:
--- name: vsliouniaev
--- name: bismarck
--- email: gianrubio@gmail.com
--  name: gianrubio
--name: prometheus-operator
-+  - name: vsliouniaev
-+  - name: bismarck
-+  - email: gianrubio@gmail.com
-+    name: gianrubio
-+name: rancher-monitoring
- sources:
--- https://github.com/coreos/kube-prometheus
--- https://github.com/coreos/prometheus-operator
--- https://coreos.com/operators/prometheus
-+  - https://github.com/coreos/kube-prometheus
-+  - https://github.com/coreos/prometheus-operator
-+  - https://coreos.com/operators/prometheus
- tillerVersion: '>=2.12.0'
- version: 8.16.1
 diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/README.md packages/rancher-monitoring/charts/README.md
 --- packages/rancher-monitoring/charts-original/README.md
 +++ packages/rancher-monitoring/charts/README.md
-@@ -2,6 +2,8 @@
+@@ -129,6 +129,38 @@
  
- Installs [prometheus-operator](https://github.com/coreos/prometheus-operator) to create/configure/manage Prometheus clusters atop Kubernetes. This chart includes multiple components and is suitable for a variety of use-cases.
- 
-+You must install the Prometheus Operator CRDs first using the `rancher-monitoring-crd` chart before installing this chart.
-+
- The default installation is intended to suit monitoring a kubernetes cluster the chart is deployed onto. It closely matches the kube-prometheus project.
- - [prometheus-operator](https://github.com/coreos/prometheus-operator)
- - [prometheus](https://prometheus.io/)
-@@ -9,6 +11,12 @@
- - [node-exporter](https://github.com/helm/charts/tree/master/stable/prometheus-node-exporter)
- - [kube-state-metrics](https://github.com/helm/charts/tree/master/stable/kube-state-metrics)
- - [grafana](https://github.com/helm/charts/tree/master/stable/grafana)
-+- [prometheus-adapter](https://github.com/helm/charts/tree/master/stable/prometheus-adapter)
-+- [rancher-pushprox](https://github.com/rancher/dev-charts/tree/master/packages/rancher-pushprox) charts to monitor internal kubernetes components for k3s, rke, and kubeAdm clusters
-+  - kube-scheduler
-+  - kube-controller-manager
-+  - kube-proxy
-+  - kube-etcd (only rke and kubeAdm)
- - service monitors to scrape internal kubernetes components
-   - kube-apiserver
-   - kube-scheduler
-@@ -136,6 +144,30 @@
- 
- The following tables list the configurable parameters of the prometheus-operator chart and their default values.
+ You may also `helm show values` on this chart's [dependencies](#dependencies) for additional options.
```

Comment:
```suggestion
 You may also run `helm show values` on this chart's [dependencies](#dependencies) for additional options.
```

Reply:
good catch, will be updated in the next commit!